xbuild-target/README.md

```diff
@@ -0,0 +1,4 @@
+# xbuild-target
+
+This is a crate that can be built via `x build` in order to simplify CI testing.
```

Comment:
`xbuild` does not support building examples?

Reply:
Unfortunately it does not.

Reply:
Mind if I get around to "quickly" hacking that in (you'd have to install `xbuild` from the `master` branch in CI) to get this going, and save you the temporary example?

Reply:
`install-action` can only install released versions of projects.